fidelius-sdk/src/main/java/org/finra/fidelius/FideliusClient.java

```diff
@@ -135,6 +130,20 @@ private  String getPrefixedName(String credentialName, Map tags) {
         }
     }
 
+    private  String getPrefixedNameForMetadata(String credentialName, Map tags) {
+        if (tags.containsKey(Constants.FID_CONTEXT_APPLICATION) && tags.containsKey(Constants.FID_CONTEXT_SDLC)) {
+            String prefixedName;
+            if (tags.containsKey(Constants.FID_CONTEXT_COMPONENT)) {
+                prefixedName = "META#" + tags.get(Constants.FID_CONTEXT_APPLICATION) + "." + tags.get(Constants.FID_CONTEXT_COMPONENT) + "." + tags.get(Constants.FID_CONTEXT_SDLC) + "." + credentialName;
```

Comment:
String.format can help clean up how the string could look instead of doing "+" a lot.